test/Concurrency/async_computed_property.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-concurrency
+// REQUIRES: concurrency
+
+func asyncFunc(_ value: String) async {}
+
+class ComputedPropertyClass {
+  var meep: String {
+    //expected-error@+1:19{{'async' call in a function that does not support concurrenct}}
```

Comment:
`concurrenct` -> `concurrency`

Reply:
Had that fix staged, forgot to apply it before pushing. Thanks.